### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./rtdata/languages -L ba,bord,childs,hist,fo,reall,bloc,alph,dof,inout,thre,makro,chang,currentry,preserv,portugues,struc,trough,vektor`

### CODE REVIEW

File: rtengine/ipgrain.cc [hunk 1]
```diff
@@ -290,7 +290,7 @@ class GrainEvaluator {
         return 32.0 * (n0 + n1 + n2 + n3);
     }
 
-    double simplex_2d_noise(double x, double y, uint32_t octaves, double persistance, double z)
+    double simplex_2d_noise(double x, double y, uint32_t octaves, double persistence, double z)
```
Comment (luzpaz) [inline rtengine/ipgrain.cc]: please review carefully

**Comment by heckflosse** [comment]
> please review carefully

this one is better

```diff
diff --git a/rtengine/ipgrain.cc b/rtengine/ipgrain.cc
index 6d2c116cd..648565036 100644
--- a/rtengine/ipgrain.cc
+++ b/rtengine/ipgrain.cc
@@ -128,7 +128,7 @@ public:
             for (int i = 0; i < W; ++i) {
                 double wx = ox + i;
                 double x = wx / wddf;
-                double noise = simplex_2d_noise(x, y, octaves, 1.0, zoom) / s;
+                double noise = simplex_2d_noise(x, y, octaves, zoom) / s;
                 lab_L[j][i] += lut_lookup(noise * strength * GRAIN_LIGHTNESS_STRENGTH_SCALE, lab_L[j][i] / 32768.f);
             }
         }
@@ -290,7 +290,7 @@ private:
         return 32.0 * (n0 + n1 + n2 + n3);
     }
 
-    double simplex_2d_noise(double x, double y, uint32_t octaves, double persistance, double z)
+    double simplex_2d_noise(double x, double y, uint32_t octaves, double z)
     {
         double total = 0;
 
```

**Comment by Floessie** [comment]
I quickly reviewed the changes and found that `persistance` isn't used.

One thing that surprises me are the missing dots ('.') after the last sentence of some tooltips. Take the first change for example. Is that intended?

Otherwise :+1: for merge.

**Comment by luzpaz** [comment]
@heckflosse see 5f3dc53

**Pull request merged**